### ISSUE TITLE
[internal/release/9.0.1xx-rc1] All legs are building fails with `11 new packages used not in baseline`

### ISSUE THREAD

**Issue by Winniexu01** [dotnet/source-build#4580]
`internal/release/9.0.1xx-rc1`: https://dev.azure.com/dnceng/internal/_build/results?buildId=2522768 (internal Microsoft link)

**Comment by mthalman** [comment]
@ellahathaway - I'm assuming this is essentially the same issue as https://github.com/dotnet/source-build/issues/4571 even though that doesn't mention prebuilts?

**Comment by ellahathaway** [comment]
> @ellahathaway - I'm assuming this is essentially the same issue as #4571 even though that doesn't mention prebuilts?

Yes - I opened #4571 before any prebuilts occurred since the changes hadn’t flowed anywhere at that time.

**Comment by mthalman** [comment]
Waiting for this fix to flow in to resolve the issue: https://github.com/dotnet/source-build-reference-packages/pull/1020

**Comment by Winniexu01** [comment]
This issue has been fixed in the [latest build](https://dev.azure.com/dnceng/internal/_build/results?buildId=2524685) (internal Microsoft link)
